### ISSUE TITLE
A custom child headline e.g for own notes, which is not pushed to jira

### ISSUE THREAD

**Issue by NimSed** [ahungry/org-jira#277]
Hi. Thanks for this great package! 
In my workflow, I use each jira ticket as a "project", as in GTD terminology, which has many sub-tasks. I'd like to be able to place these sub-tasks right under the org item containing the ticket, but still keep them private (=local, only in the org file). Right now doing so causes problems and most of the times interrupts updating/refreshing. 
Is such a feature already implemented and I'm missing it? Otherwise, is there any possible workaround?

**Comment by ahungry** [comment]
I'm not sure - if anyone else is, feel free to chime in.

**Comment by NimSed** [comment]
Sorry for the very late reply.
Since I have migrated away from using sub-tasks, my notes might not be quite relevant any more:
- I just went ahead and added my "tasks" as org headings under the JIRA issues in my org file, and they seem to be OK: org-jira apparently does not try to push them to JIRA servers.
- Just a few minutes ago, I submitted this PR: https://github.com/ahungry/org-jira/pull/300
  What it does is to allow for moving all org-jira issues to any arbitrary org file and keeping them in sync. Without this PR it is close to impossible to do maintain such a configuration.

Hope this helps.
